reddit_gold/templates/snoovatar.html

```diff
@@ -64,6 +64,14 @@
       <div class="username">${thing.username}</div>
     %endif
     <p>${_md('view and discuss snoovatars in /r/snoovatars')}</p>
+    %if c.user.gold and c.user.name != thing.username:
+      ${_md('create your own snoovatar [here](/user/%(username)s/snoo)' % dict(username=c.user.name))}
```

Comment:
So, subtle problem here that I just noticed: This will break translations for this string, because the interpolation (%) happens before _ gets called, which means that when the translation function goes to look up the string, it will always be different based on user.

I think what we want instead, then, is `_mdf`, which allows formatting to happen after translation lookup. Like this:

```mako
${_mdf('create your own snoovatar [here](/user/%(username)s/snoo)', username=c.user.name)}
```

This should be safe and format properly. Was talking to @JordanMilne about this and a lot of this is confusing, so we might put together a cheat sheet of templating functions and when to use them.